### DISCUSSION TITLE
Docker issue

### DISCUSSION THREAD

**Discussion by Physics-Morris** [alshedivat/al-folio#2292] [Q&A] [answered]
I ran into following error when I try to execute the command docker compose up.
I try to delete the container and create a new one, but it doesn't help. Does anyone know what happened? TKS

================================================================================
Attaching to jekyll-1
jekyll-1  | /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/resolver.rb:332:in `raise_not_found!': Could not find gem 'jekyll-diagrams' in locally installed gems. (Bundler::GemNotFound)
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/resolver.rb:392:in `block in prepare_dependencies'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/resolver.rb:377:in `each'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/resolver.rb:377:in `map'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/resolver.rb:377:in `prepare_dependencies'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/resolver.rb:61:in `setup_solver'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/resolver.rb:28:in `start'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/definition.rb:600:in `start_resolution'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/definition.rb:311:in `resolve'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/definition.rb:553:in `materialize'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/definition.rb:203:in `specs'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/definition.rb:270:in `specs_for'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler/runtime.rb:18:in `setup'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/bundler-2.5.4/lib/bundler.rb:162:in `setup'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/jekyll-4.3.3/lib/jekyll/plugin_manager.rb:52:in `require_from_bundler'
jekyll-1  |     from /var/lib/gems/3.0.0/gems/jekyll-4.3.3/exe/jekyll:11:in `<top (required)>'
jekyll-1  |     from /usr/local/bin/jekyll:25:in `load'
jekyll-1  |     from /usr/local/bin/jekyll:25:in `<main>'
================================================================================

**Comment by george-gca** [comment, marked as answer]
You are probably using a very old version of al-folio. I'd suggest you to update your repository.
